process/project-lifecycle.md

```diff
@@ -172,13 +172,23 @@ See [Submission Process](#submission-process) below and the [Archive application
 
 ### Project creation or change of lifecycle stage
 
-For initiating the creation of a new project or for requesting a change of a project's lifecycle stage, an application must be submitted to the TAC. To this end, a maintainer of the project (or anyone in the case of Archived projects) creates a PR in this repository with the following changes:
+For initiating the creation of a new initiative (project or SIG), an application must be submitted to an OpenSSF WG (or directly to the TAC if it wants to be a "top-level project" under the TAC). For submission to a WG, the project must be within scope of the WG. Of course, the WG must approve adding the initiative.
+
+If it's a pre-existing work (code, documentation, etc.), there are a few extra steps. We do *not* require copyright assignment, and future results will continue to be usable by its submitters. However, the submitters must be willing to license the work per the [OpenSSF Charter](https://openssf.org/about/charter/), that is, it must use a pre-approved license for its type or receive OpenSSF Governing Board approval. Pre-existing projects must notify the OpenSSF of any intellectual property of the project such as the existing code/documentation, trademarks, patents, and domain registrations. Please do this by sending an email to `operations` at `openssf.org`, so that we can perform a legal review. The licensing must be settled, trademarks and domains of the project itself must be reassigned to the OpenSSF, and patent issues must be dealt with before the project can join. This legal review should happen in parallel with other steps, to prevent slowing things down. Note: We generally expect that the current project leads are in favor of joining the OpenSSF. Even when it's legal to do so, we typically avoid forking whole projects, because our goal is to *collaborate* to produce great results.
+
+If a WG approves adding the project, the WG (usually a WG lead) must notify the TAC and give the TAC adequate time to object (e.g., at least 2 weeks). If TAC doesn't object, and the legal review approves, it's an OpenSSF project. A TAC member may ask for further review, at which point it becomes a decision item for the TAC.
+
+Once it's an OpenSSF project, there must be a location of its material (typically on GitHub, though some early drafts may start on other systems such as GitHub). The WG's README file must be modified to identify and link to that project's entry page, and the project's entry page must be modified to identify and link t its parent its WG (or the TAC if it's a top-level project).
```

Comment:
GitHub twice here, should the second one be GitLab?
```suggestion
Once it's an OpenSSF project, there must be a location of its material (typically on GitHub, though some early drafts may start on other systems such as GitLab). The WG's README file must be modified to identify and link to that project's entry page, and the project's entry page must be modified to identify and link to its parent its WG (or the TAC if it's a top-level project).
```

Reply:
Ack! I meant Google (specifically Google docs). Will fix.